robottelo/config/base.py

```diff
@@ -477,10 +477,10 @@ def read(self, reader):
     def validate(self):
         """Validate docker settings."""
         validation_errors = []
-        if not self.docker_image:
-            validation_errors.append('[docker] docker_image option must be provided or enabled.')
-        if not self.external_registry_1:
-            validation_errors.append('[docker] external_registry_1 option must be provided.')
+        if not all(self.__dict__.values()):
```

Comment:
Why the change to making all of the fields required?

Though this may be accurate for the tests, I'd like to hold on enforcing that until the facade layer is removed, and settings file validation is tied to test collection/fixture execution.

Reply:
Ok, removed this change.